.travis.yml

```diff
@@ -22,7 +22,7 @@ services:
 
 language: scala
 scala:
-  - 2.12.18
+  - 2.13.11
```

Comment:
we'll need to have both a scala 2.12 and a scala 2.13 distribution installed in travis (so the cross builds can run).  Refer to travis docs for how to do this.